tests/fixtures/data/organism.py

```diff
@@ -0,0 +1,90 @@
+ORGANISM_HOSTS = [
```

Comment:
Thanks for making this effort. It is going to make testing more satifying.

Make the spacing consistent. It looks like `ruff` is not configured to run in your IDE.